internal/scti/handlers.go

```diff
@@ -62,20 +63,53 @@ var (
 	// Metrics are all per-log (label "origin"), but may also be
 	// per-entrypoint (label "ep") or per-return-code (label "rc").
 	once             sync.Once
-	knownLogs        monitoring.Gauge     // origin => value (always 1.0)
-	lastSCTTimestamp monitoring.Gauge     // origin => value
-	reqsCounter      monitoring.Counter   // origin, ep => value
-	rspsCounter      monitoring.Counter   // origin, ep, rc => value
-	rspLatency       monitoring.Histogram // origin, ep, rc => value
+	knownLogs        *prometheus.GaugeVec     // origin => value (always 1.0)
```

Comment:
Could just remove this one, I suspect?

Reply:
I've left it on purpose: this will allow to build global dashboard with all the log names for instance, if multiple stacks exist. It's a nice way to extract the origin, and protects us from bugs like nan values on some metrics because no request is flying around.